### ISSUE TITLE
Use markdown code regions in completion descriptions

### ISSUE THREAD

**Issue by panoply** [wkillerud/vscode-scss#57]
Hi @wkillerud, firstly, thank you for the efforts and work you've done here by expanding upon vscode-scss. 

I've got a minor feature request here. When generating completion descriptions, the current logic works like this. Take the follow map:

```scss
/// Controls flex directions to use
///
$flex-direction: (
  row: row,
  col: column,
  // row-rev: row-reverse,
  // col-rev: column-reverse
);
```

The above would generate the following in completion descriptions:

<img width="904" alt="Screenshot 2023-06-06 at 14 44 59" src="https://github.com/wkillerud/vscode-scss/assets/7041324/6c4a9592-0f89-4d8b-8563-c63018751f74">

After quickly having a look at the source, [`getLimitedString`](https://github.com/wkillerud/vscode-scss/blob/main/server/src/utils/string.ts#L42-L53)  is handling the documentation generated for variables. It's my understanding that markdown code regions can also apply to completion descriptions in LSP and it would be nice to have the identical generation applied to completion items as with hovers. 

For example, the above code sample in hovers:

<img width="353" alt="Screenshot 2023-06-06 at 14 53 05" src="https://github.com/wkillerud/vscode-scss/assets/7041324/4971181f-e482-4c21-8fb5-df03fd43f805">

Is this is the _intended_ behavior for Some Sass or is there a particular reason why markdown code blocks are excluded in completions but not hovers? 

Thanks.

**Comment by wkillerud** [comment]
Hi @panoply

> Is this is the intended behavior for Some Sass or is there a particular reason why markdown code blocks are excluded in completions but not hovers?

It's been a while since I last poked around there, and I think some of the decisions are pretty old. For instance, `getLimitedString` has `git blame` going back seven years. Maybe there were some limitations in VS Code back then that are no longer relevant.

I'm certainly not opposed to have a richer experience in completions as well, if it's technically possible. I see TypeScript completions include `@example` with syntax highlighting, so should be doable.

**Comment by wkillerud** [comment]
Updated suggestions format shipped in [`2.13.1`](https://github.com/wkillerud/vscode-scss/releases/tag/2.13.1). There's a known limitation in VS Code for color suggestions, where there's no support for Markdown (https://github.com/microsoft/vscode/issues/185055). I'm working on landing support for that in a future version.
